tests/cli/formatters/test_config_formatters.py

```diff
@@ -220,62 +152,43 @@ def test_format_time_9999h_59m(self) -> None:
 
 
 class TestQuotaFormatter:
-    def test_output(self) -> None:
+    def test_output(self, rich_cmp: RichCmp) -> None:
         quota = _Quota(
             total_gpu_run_time_minutes=321,
             total_non_gpu_run_time_minutes=123,
         )
         out = QuotaFormatter()(quota)
-        assert out == "\n".join(
-            [f"{bold_start}GPU:{bold_end} 321m", f"{bold_start}CPU:{bold_end} 123m"]
-        )
+        rich_cmp(out)
 
-    def test_output_no_quota(self) -> None:
+    def test_output_no_quota(self, rich_cmp: RichCmp) -> None:
         quota = _Quota(
             total_non_gpu_run_time_minutes=None, total_gpu_run_time_minutes=None
         )
         out = QuotaFormatter()(quota)
-        assert out == "\n".join(
-            [
-                f"{bold_start}GPU:{bold_end} infinity",
-                f"{bold_start}CPU:{bold_end} infinity",
-            ]
-        )
+        rich_cmp(out)
 
-    def test_output_only_gpu(self) -> None:
+    def test_output_only_gpu(self, rich_cmp: RichCmp) -> None:
         quota = _Quota(
             total_gpu_run_time_minutes=9923, total_non_gpu_run_time_minutes=None
         )
         out = QuotaFormatter()(quota)
-        assert out == "\n".join(
-            [
-                f"{bold_start}GPU:{bold_end} 9923m",
-                f"{bold_start}CPU:{bold_end} infinity",
-            ]
-        )
+        rich_cmp(out)
```

Comment:
An assertion example.

File is auto-calculated from test module, test class, and test function.